tests/CMakeLists.txt

```
@@ -3,11 +3,53 @@ set(PHPUNIT "${CDash_SOURCE_DIR}/vendor/bin/phpunit")
 
 function(add_vue_test TestName)
     add_test(
-        NAME ${TestName}
-        COMMAND "node_modules/.bin/jest" "tests/${TestName}.spec.js"
-        WORKING_DIRECTORY "${CDash_SOURCE_DIR}"
-        )
-endfunction(add_vue_test)
+      NAME ${TestName}
+      COMMAND "node_modules/.bin/jest" "tests/${TestName}.spec.js"
+      WORKING_DIRECTORY "${CDash_SOURCE_DIR}"
+    )
+endfunction()
+
+if(DEFINED $ENV{APP_URL})
```

I'm pretty sure that here we actually don't want to dereference the variable
```suggestion
if(DEFINED ENV{APP_URL})
```

---

Looks like you're right, although both seem to work.  https://cmake.org/cmake/help/latest/variable/ENV.html

That's certainly not confusing at all...